docs/antrea-l7-network-policy.md

```diff
@@ -200,6 +201,80 @@ spec:
         - http: {}       # automatically dropped, and subsequent rules will not be considered.
 ```
 
+### Logs
+
+Layer 7 traffic that matches the NetworkPolicy will be logged in an event
+triggered log file (`/var/log/antrea/networkpolicy/l7engine/eve-YEAR-MONTH-DAY.json`).
+The event type for this log is `alert`. If `enableLogging` is set for the rule,
```

Comment:
You mean the dropped request will be logged regardless of whether enableLogging is enabled? Is it configurable?